.github/workflows/analysis_workflow.yml

```diff
@@ -14,11 +14,33 @@ on:
     - "**/*.md"
 
 jobs:
+
+  run_linting_checks:
+    name: Linting checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3.3.0
+
+      - name: Install linters
+        run: |
+          python3 build_tooling/format.py --install-tools
+
+      - name: Lint Python
+        if: always()
```

Comment:
Why is `if: always()` needed? `Install linters` above doesn't use it.

Reply:
So that we give the linting output for C++ even if the Python linting fails. People will probably want to see all the problems at once.